cli/tui/ongoing/markdown_projection.go

```
@@ -0,0 +1,253 @@
+				RowIndex:          firstDifferentRow(fullRows, candidateRows),
+			},
+		}
+	}
+	return markdownProjectionResult{
+		PromotedRows:     candidateRows,
+		VolatileRows:     append([]string(nil), fullRows[len(candidateRows):]...),
+		PromotedBoundary: input.PromotedBoundary + candidateBoundary,
+	}
+}
+
+type plainMarkdownRenderer struct{}
```

golangci-lint (2.12.2)

[error] type plainMarkdownRenderer is unused

(unused)